mne/viz/epochs.py

```diff
@@ -95,6 +99,13 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
     scale_vmin = True if vmin is None else False
     scale_vmax = True if vmax is None else False
     vmin, vmax = _setup_vmin_vmax(data, vmin, vmax)
+    if logscale:
+        if (data >= 0).all():
```

Comment:
Just ensuring the sum > 0 doesn't necessarily mean that all the entries can be converted to log, right?